flixel/FlxG.hx

```diff
@@ -309,6 +309,10 @@ class FlxG
 	 */
 	public static var plugins(default, null):PluginFrontEnd;
 
+	/**
+	 * Whether the game is running on HTML5.
+	 */
+	 public static var onHTML5:Bool = #if html5 true #else false;
```

Comment:
1. this should be readonly, look at `FlxG.onMobile` as a guide

Reply:
ok i see i go do that

Reply:
This won't compile because you never created the getter, like this
https://github.com/HaxeFlixel/flixel/blob/dev/flixel/FlxG.hx#L768-L778

Reply:
huh? i dont need no getter i do #if html5 true #else false; #end

Reply:
you need a getter if you have `(get, never)`

you can also just make it inline
```public static inline var onHTML5:Bool = #if html5 true #else false; #end```
also note that you need `#end`

Reply:
also there seems to be a space before public, please remove that

Reply:
ah see im not good with var stuff ig